src/Evolution/Systems/Cce/NpValues.hpp

```diff
@@ -0,0 +1,63 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/SpinWeighted.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/Cce/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class ComplexDataVector;
+/// \endcond
+
+namespace Cce {
+
+template <typename Tag>
+struct VolumeWeyl;
+
+/*!
+ * \brief Compute the Weyl scalar Psi0 in the volume according to a standard set
+ * of Newman-Penrose vectors.
+ *
+ * \details The Bondi forms of the Newman-Penrose vectors that are needed for
+ * Psi0 are:
+ *
+ * \f{align}{
+ * \mathbf{l} &= \partial_r / \sqrt{2}\\
+ * \mathbf{m} &= \frac{-1}{2 r} \left(\sqrt{1 + K} q^A \partial_A -
+ *   \frac{J}{\sqrt{1 + K}}\bar{q}^A \partial_A \right)
+ * \f}
+ *
+ * Then, we may compute \f$\psi_0 =  l^\alpha m^\beta l^\mu m^\nu C_{\alpha
```

Comment:
is it standard to use `\Psi` instead of `\psi` for the NP scalars?

Reply:
I've seen both, but I'm happy to make them all uppercase.